trackers_integration/tests/test_mantis.py

```diff
@@ -0,0 +1,193 @@
+# pylint: disable=attribute-defined-outside-init
+import os
+import unittest
+
+from django.utils import timezone
+
+from tcms.core.contrib.linkreference.models import LinkReference
+from tcms.issuetracker.mantis import Mantis, MantisAPI
+from tcms.rpc.tests.utils import APITestCase
+from tcms.testcases.models import BugSystem
+from tcms.tests.factories import ComponentFactory, TestExecutionFactory
+
+
+class TestMantisIntegration(APITestCase):
+    existing_bug_id = 1
+    existing_bug_url = "http://localhost/rest/api/issues/1"
```

Comment:
Do you know what's the situation with Mantis containers ? Is there a container (preferably from official sources) that can be used for testing or do we need to build one ourselves ?

Reply:
Seems, you have found one. #17